qartod/parad/parad_qartod_climatology_test_values.csv

```diff
@@ -0,0 +1,11 @@
+subsite,node,sensor,stream,parameters,climatologyTable,source,notes
+CE01ISSP,SP001,10-PARADJ000,parad_j_cspp_instrument,"{'inp': 'parad_j_par_counts_output', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE01ISSP-SP001-10-PARADJ000-par.csv,Climatology based on all data collected through 2022-01-01.,
+CE01ISSP,SP001,10-PARADJ000,parad_j_cspp_instrument_recovered,"{'inp': 'parad_j_par_counts_output', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE01ISSP-SP001-10-PARADJ000-par.csv,Climatology based on all data collected through 2022-01-01.,
+CE02SHSP,SP001,09-PARADJ000,parad_j_cspp_instrument,"{'inp': 'parad_j_par_counts_output', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE01ISSP-SP001-09-PARADJ000-par.csv,Climatology based on all data collected through 2022-01-01.,
+CE02SHSP,SP001,09-PARADJ000,parad_j_cspp_instrument_recovered,"{'inp': 'parad_j_par_counts_output', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE01ISSP-SP001-09-PARADJ000-par.csv,Climatology based on all data collected through 2022-01-01.,
+CE06ISSP,SP001,10-PARADJ000,parad_j_cspp_instrument,"{'inp': 'parad_j_par_counts_output', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE01ISSP-SP001-10-PARADJ000-par.csv,Climatology based on all data collected through 2022-01-01.,
+CE06ISSP,SP001,10-PARADJ000,parad_j_cspp_instrument_recovered,"{'inp': 'parad_j_par_counts_output', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE01ISSP-SP001-10-PARADJ000-par.csv,Climatology based on all data collected through 2022-01-01.,
+CE07SHSP,SP001,09-PARADJ000,parad_j_cspp_instrument,"{'inp': 'parad_j_par_counts_output', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE01ISSP-SP001-09-PARADJ000-par.csv,Climatology based on all data collected through 2022-01-01.,
+CE07SHSP,SP001,09-PARADJ000,parad_j_cspp_instrument_recovered,"{'inp': 'parad_j_par_counts_output', 'tinp': 'time', 'zinp': 'None'}",climatology_tables/CE01ISSP-SP001-09-PARADJ000-par.csv,Climatology based on all data collected through 2022-01-01.,
```

Comment:
Are these supposed to have a depth parameter/input?

Reply:
Yes, see newly updated tables.